src/routes/chat/[id]/+page.svelte

```diff
@@ -103,7 +103,14 @@
 		loading = false
 	}
 
-	let chatImg = $chats.chats.get($page.params.id)?.users[0].avatar
+	let chatImg: string | undefined = undefined
+	if ($chats.chats.get($page.params.id)?.users.length === 2) {
+		$chats.chats.get($page.params.id)?.users.forEach((member) => {
+			if (member.address !== $walletStore.wallet?.address) {
+				chatImg = member.avatar
+			}
+		})
+	}
```

Comment:
Thanks, this is quite correct. However, just FYI I would write it this way (not tested)

```suggestion
let chatImg: string | undefined = undefined
$: if ($chats.chats.get($page.params.id)?.users.length === 2) {
    const otherUser = $chats.chats.get($page.params.id)?.users.find(m => m.address !== $walletStore.wallet?.address)
    chatImg = otherUser?.avatar
}
```

Reply:
You can also to it all in one line but that would be really hard to read (even this is).

Reply:
Also it may be useful to actually have the `otherUser` be available because in addition to the chat image being `otherUser.avatar`, the name of the chat also is  `otherUser.name`.

Reply:
@vojtechsimetka 
Cool, thank you! I applied your suggestion manually because I wanted to test first 😊
Yes, I think initially we were typing in the chat name but if we're only doing 1-1 chats for now, we should use otherUser. I'll look into that too.
Thanks!